themes/autumn.toml

```diff
@@ -0,0 +1,33 @@
+# Added by: ArmanDris
+# From: https://github.com/yorickpeterse/Autumn.vim/blob/master/colors/autumn.vim
+
+[colors]
+# Colors for the terminal
```

Comment:
```suggestion
```

No need to add an empty table here and the comment doesn't really say anything worth saying.